packages/integration-tests/index.js

```diff
@@ -325,14 +325,14 @@ const loadCollection = async (params = {}) =>
  * @param {string} bucketName - S3 internal bucket name
  * @param {Array} collections - List of collections to delete
  * @param {string} postfix - string that was appended to collection name
- * @returns {Promise.<number>} number of deleted collections
+ * @returns {Array<Object>} a list of the http responses
  */
 async function deleteCollections(stackName, bucketName, collections, postfix) {
   // setProcessEnvironment is not needed by this function, but other code
   // depends on this undocumented side effect
   setProcessEnvironment(stackName, bucketName);
 
-  await Promise.all(
+  return await Promise.all(
```

Comment:
This caused a lot of grief. Previously it was returning `collections.length;`, which was always the length of the input collections. It looked like it deleted the appropriate rows but didn't necessarily.

Reply:
:thumbsup: nice catch.